veg2hab/mozaiek.py

```diff
@@ -1,14 +1,17 @@
-import json
 import logging
-from collections import defaultdict
+from collections import defaultdict, namedtuple
 from numbers import Number
-from typing import ClassVar, Dict, List, Optional, Tuple, Union
+from typing import ClassVar, Dict, List, Optional, Set, Tuple, Union
 
 import geopandas as gpd
+import pandas as pd
 from pydantic import BaseModel, Field, PrivateAttr
 
 from veg2hab.enums import Kwaliteit, MaybeBoolean
 from veg2hab.io.common import Interface
+from veg2hab.vegetatietypen import SBB, VvN
+
+MozkPercTuple = namedtuple("MozkPercTuple", ["habtype", "kwaliteit", "percentage"])
```

Comment:
Is NamedTuple uit Typing al onderdeel van python3.7?

Reply:
Ik draai 3.7 dus denk het wel dan